layouts/partials/header.html

```diff
@@ -1,14 +1,21 @@
-<script
-  async
-  src="https://www.googletagmanager.com/gtag/js?id=UA-101654320-1"
-></script>
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-6VPFYD9902"></script>
```

Comment:
I know next to nothing about this, but I'm surprised we need both tags? I assumed we'd replace the old `UA-101654320-1` version with the `G-6VPFYD9902` version. I guess we need `UA-101654320-4` too?

Otherwise LGTM 🤷‍♀️

Reply:
Okay, thanks!

Reply:
Response, FYI
> Yes we will need both tags. One is for a new iteration of Analytics called GA4 and one is for the pre-existing universal Analytics property. At this time, It is recommended to have both property types for tracking purposes.